site/_posts/2021-04-01-why-user-multi-factor-authentication.md

```diff
@@ -0,0 +1,129 @@
+
+## When should I require multiple factors of authentication?
+
+There are times and circumstances where you, as a system developer, should require an increased level assurance about who is logging in. Often the type of account is relevant, other times it is the requested access, and sometimes it is legal requirements or corporate policies.
+
+### Administrative accounts
+
+Privileged accounts with higher levels of access should use MFA. 
+
+These administrator or operator accounts may wreak havoc if misused or compromised. 
+
+To address this, require MFA on all admin accounts. Require providing additional factors regularly, perhaps at every login. In extremely sensitive systems, all changes to a system could require multiple factors.
+
+### High value accounts
+
+There are also many high value accounts where MFA helps to prevent account compromise. These user accounts don't possess elevated privileges within a system, unlike the aforementioned admin accounts. Instead, they control data or allow actions to be taken with real world consequences.
+
+One example is an online bank account. You don't want users to lose their account access and later learn someone drained their savings. 
+
+Another example is an email account. Beyond the private information often present in inboxes, these represent a risk to many other systems. Password reset solutions often send an email; compromise of an email account means risk to all such accounts of this user.
+
+### Risky actions
+
+When a user has already authenticated, but is performing a dangerous action, MFA provides extra security. This is also known as "step up auth", because the additional factor is required at the moment the risky action is requested. Examples may include:
+
+* Changing a password or username
+* Spending money
+* Resetting a password
+* Creating a new user with elevated privileges
+* Changing system settings
+* Modifying configuration impacting other factors, such as a user's email or phone number
+
+When an action like this is requested, the system needs certainty about the requester. These actions can be, and often are, legitimate. But they can also be used by someone with illicit access to the user's account to solidify the takeover.
+
+By using step up auth, you can help mitigate the damage of a compromised account. An attacker may be able to access the account and read data from it, but if they don't have the additional factor, they won't be able to undertake actions which can cause more damage.
+
+### Legal or organizational policies
+
+If your application is used by certain organizations or stores personally identifiable information, you may need to require multi-factor authentication for some or all accounts. As part of the NIST risk management framework, for example, Authenticator Assurance Level 2 requires: ["proof of possession and control of two different authentication factors is required..."](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-63-3.pdf). 
```

Comment:
Is this period places properly after the link?  I think so, but...time to get out my Strunk & White book...

Reply:
What did S&W say? :)

Reply:
Alas, it's buried in my bins of stuff that I "have to keep" 😆 


I think that it doesn't cover at the condition of a link, but they are using four `....`
[link](https://www.uhv.edu/university-college/student-success-center/resources/e-p/ellipsis-use-in-mla-style/#:~:text=To%20form%20an%20ellipsis%20in%20MLA%20when%20the%20omitted%20material,comes%20after%20the%20parenthetical%20citation)

Somehow, I think that we will be ok either way 😄 

---
The ellipsis at the End of a Sentence
To form an ellipsis in MLA when the omitted material appears at the end of your sentence, type the ellipsis with three periods with space before each period and include the closing quotation marks immediately after the third period. Note that the period to end your sentence comes after the parenthetical citation.

Original: It also requires you to listen to emotions, not just to facts, and to understand the players – speaker and audience –
intended and otherwise.

```
Parts Omitted:Bryan, Cameron, and Allen emphasize that listening “. . . requires you to listen to emotions . . .” (147).

If your sentence ends in omitted material and a parenthetical citation isn’t used, then four periods are included in the ellipsis, with no space before the first period or after the last one.

Ex. Winston Churchill in a speech that became known as We Shall Fight on the Beaches declared, “We shall go on to the end,
we shall fight in France . . . we shall fight on the beaches, we shall fight on the landing grounds, we shall fight in the fields and
in the streets, we shall fight in the hills; we shall never surrender. . . ."
```